packages/riverpod_generator/lib/src/templates/family.dart

```diff
@@ -26,10 +26,11 @@ class ${data.familyName} extends Family<${data.exposedValueDisplayType}> {
   }
 
   @override
-  List<ProviderOrFamily>? get allTransitiveDependencies => null;
+  List<ProviderOrFamily>? get allTransitiveDependencies => 
```

Comment:
Also I'd store the list (and `dependencies`too in a static final variable. 
To cache the list instance

Reply:
Finished putting them in static final variables, however, to do the transitive dependencies we would have to parse all of the data before writing any of the providers. This is doable, but I saw this comment which made me question if we could use it:
```dart
 @override
  Stream<String> generateForAnnotatedElement(
    Element element,
    ConstantReader annotation,
    BuildStep buildStep,
  ) async* {
    // implemented for source_gen_test – otherwise unused
```
So for source_gen_test will this cause issues, since it considers the annotations one at a time, and not all together?